cloudify_agent/api/pm/base.py

```diff
@@ -220,15 +220,8 @@ def __init__(self, logger=None, **params):
         # Mandatory parameters
         self.validate_mandatory()
         self.rest_host = params['rest_host']
-        if isinstance(params['broker_ip'], basestring):
```

Comment:
this now always gets a list, because the commandline param is split inside of the click parsing callback